tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -225,6 +224,11 @@ sub add_junit_log {
 sub upload_guest_assets {
     my $self = shift;
 
+    if (get_var('CASEDIR')) {
+        record_info('Skip uploading guest assets', 'Guest assets will only be uploaded with test codes in git master branch.');
```

Comment:
This will block us from validating future PRs to upload_guest_assets directly on OSD.
If I understand correctly, the check of `CASEDIR` is not necessary for deciding upload or not. Maybe better to keep original code?

Reply:
I noticed that allowing a test with a PR to upload guest assets always brought troubles.  

- Tests fail if UPLOAD_GUEST_ASSETS=0 is not set explicitly in a clone job command line() as half of our tests set this setting 1 by default, especially for people from other team who would like to verify if his change breaks our test. 
- Guest assets were uploaded unintentionally by clone jobs without UPLOAD_GUEST_ASSETS=0. I guess most of our team members were not aware it, let alone others. 

I have had this thought to fixe it before this PR but have not carried it out. This is a chance to both remove SCC_URL and stop uploading guest assets from a clone job.

It does stop verifying the changes to upload guest assets in one's PR. A work around is to comment out this statement in their PR. This needs will be rare. 

I am inclined to disallow upload assets from a clone job with the cons and pros listed above.

Reply:
I see. It's fine to do it if it is useful. 

Just a suggestion -- it will be useful to add motivation of such change in PR description(and even better in ticket). It is obviously another purpose in addition to "The purpose is to remove the usage of SCC_URL from test scripts outside of boot_from_pxe.pm." ;-)

Reply:
I see. It's fine to do it if it is useful. 

Just a suggestion -- it will be useful to add motivation of such change in PR description(and even better in ticket). It is obviously another purpose in addition to "The purpose is to remove the usage of SCC_URL from test scripts outside of boot_from_pxe.pm." ;-)

Reply:
Yes ~ I did not realized this until you asked ~ have add it to PR description.